### PR TITLE
WIP: Add circle CI support

### PR DESCRIPTION
**Pull request by nickrobison**
Initial support for building on Circle CI.

Still a work in progress, only the build phase works.

### CODE REVIEW

**Comment by luispuerto** [comment]
Sorry, I couldn't resist: 
 

Really good work!!! 👍

File: .circleci/after_script.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+###########################################################################
+#    homebrew-osgeo4mac travis ci - after_script.sh
+#    ---------------------
+#    Date                 : Dec 2016
+#    Copyright            : (C) 2016 by Boundless Spatial, Inc.
+#    Author               : Larry Shaffer
```
Comment (3nids) [inline .circleci/after_script.sh]: get your name down there, you deserve it!

**Comment by fjperini** [comment]
@nickrobison great work!

**Comment by 3nids** [comment]
@nickrobison what's the status here? why closing?

**Comment by nickrobison** [comment]
@3nids I have the CircleCI integration working, I closed this in order to rebase and merge a clean PR.

**Pull request closed without merging**